h5p.classes.php

```diff
@@ -1245,7 +1258,7 @@ private function arrayCopy(array $array) {
     $result = array();
     foreach ($array as $key => $val) {
       if (is_array($val)) {
-        $result[$key] = arrayCopy($val);
```

Comment:
same concept, PHP is forgiving but you are calling a static function and need to use self::